.github/workflows/ci.yml

```diff
@@ -216,10 +216,10 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        node: [16, 18, 20]
-        platform: [linux-x64, linux-arm, macos-x64, macos-arm, windows-x64]
-        reuse-v8-context: [true, false]
-        server: [cli] # FIXME: Add 'cloud'
+        node: [ 16, 18, 20 ]
+        platform: [ linux-x64, linux-arm, macos-x64, macos-arm, windows-x64 ]
+        reuse-v8-context: [ true, false ]
+        server: [ cli ] # FIXME: Add 'cloud'
```

Comment:
My IDE *insists* that this is the proper formatting. I can change it back if anyone thinks it matters.

Reply:
And mine insists it is not… 🫤

I believe prettier is configured to format that file. Just do `npm run format`, and it should fix what your IDE did.